about/accounts.md

```diff
@@ -13,6 +13,23 @@ New Users can request an account by [opening an issue] on Github.
 
 [opening an issue]: https://github.com/ArjunaCluster/ArjunaUsers/issues/
 
+## Acceptable Use Policy
```

Comment:
Give me something at the very front that's like "As long as you don't break any laws, only use arjuna for research, don't share accounts or passwords, and don't run things on the headnode, you're probably fine."